src/components/Logo/Logo.tsx

```diff
@@ -10,12 +10,13 @@ import './Logo.scss';
 const b = block('logo');
 
 export const Logo: React.FC<
-    LogoProps & {compact?: boolean; buttonClassName?: string; buttonWrapperClassName?: string}
```

Comment:
This is breaking change. Please retarget your PR to branch `v3`. Also message for commits with breaking changes should be starts with `feat!`